package.json

```diff
@@ -53,9 +54,11 @@
     "eslint-plugin-import": "^2.27.5",
     "eslint-plugin-sort-exports": "^0.8.0",
     "jest": "^28.1.1",
+    "jsonwebtoken": "^9.0.0",
```

Comment:
Before [the suggested change above to the type of jwtOptions](https://github.com/PhilanthropyDataCommons/service/pull/124#discussion_r1128674483) there was only a `type` import from `jsonwebtoken`, right? So it would come from `@types/jsonwebtoken` and all the `@types` are only `devDependencies`. Granted, `jsonwebtoken` is a transitive dependency of one of our `dependencies` (`espress-jwt`), so it does end up in the `dependencies` in `package-lock.json`.

Now with [the suggested change above to the type of jwtOptions](https://github.com/PhilanthropyDataCommons/service/pull/124#discussion_r1128674483) there is no direct dependency on `@types/jsonwebtoken` nor `jsonwebtoken` in `src/auth/jwtOptions.ts`, but there is one import in `src/middleware/jwtTokenChecker.ts` of the `JwtPayload` type which is needed at compile time. Therefore `@types/jsonwebtoken` can stay at compile time (`devDependencies` in package.json) but we don't need a reference to plain `jsonwebtoken` at all in the `package.json` It might still arrive in `package-lock.json` in non-dev `dependencies` due to being a transitive dependency, though.

Reply:
I'll look at this more deeply later, but in general the policy is: if you `import` (not `import type`) from a dependency in non-test code, it goes in `dependencies`, even if it's already a transitive dependency; if you `import` from a dependency in test code, or `import type` from a dependency in non-test code, it goes in `devDependencies`; if you never import it, then leave it to your direct dependencies to bring it in as a transitive dependency (except in some special cases).

I think you already know that, but I mention it just in case; more likely, what's happening is that there's been a bit of thrash on this branch and we're talking about different states of the world!

Reply:
It sounds like we're on the same page. I think this is resolved. If not, we can revisit.